styleguide/derek/_partials/solutions/header-largeIMAC.ejs

```diff
@@ -1,18 +1,20 @@
-<!--Header Large-->
-<div class="imacBanner-container large">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-7">
-        <div class="imacBanner-text">
-          <h2 class="imacBanner-headline">3,000 cloud experts working for you 24/7/365?<br/>
-          Sounds like you may need a bigger office.</h2>
-        </div>
+<!--Header Long-->
+<div class="imacBanner large">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="col-sm-7 imacBanner-textContainer">
+        <h2 class="imacBanner-headline imacBanner-headlineLarge">we guarantee a 99.99% openstack API uptime.<br/>And yes, that .01% does keep us up at night</h2>
+        <span class="line left-top"></span>
+        <span class="line left-bottom"></span>
+        <span class="line right-bottom"></span>
       </div>
     </div>
-    <div class="row">
-      <div class="col-sm-4">
-        <a class="button banner">Choose A Plan</a>
-      </div>
+    <div class="imacBanner-buttonRow">
+      <a class="button imacBanner-button">Read Our Case Studies  <i class="fa fa-angle-right imacBanner-buttonIcon"></i></a>
     </div>
   </div>
 </div>
+
+<script type="text/javascript">
+	$(".imacBanner-headlineLarge").fitText(1.1, { maxFontSize: '45px' });
```

Comment:
Also, it seems that every one of these is the same, could we not just apply this script to the `imacBanner-headline` class instead of using these non-styled class names?